diff -Nur -x '*.orig' -x '*.rej' e-smith-ibays-1.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/ibays.pm mezzanine_patched_e-smith-ibays-1.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/ibays.pm
--- e-smith-ibays-1.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/ibays.pm	2005-10-06 17:52:29.000000000 -0400
+++ mezzanine_patched_e-smith-ibays-1.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/ibays.pm	2007-08-30 11:32:19.000000000 -0400
@@ -482,7 +482,8 @@
 sub print_save_or_add_button {
     my ($self) = @_;
 
-    if ($self->cgi->param("action") eq "modify") {
+    my $action = $self->cgi->param("action") || '';
+    if ($action eq "modify") {
         $self->print_button("SAVE");
     } else {
         $self->print_button("ADD");
