diff -Nur -x '*.orig' -x '*.rej' e-smith-ibays-1.2.0/root/etc/e-smith/web/functions/ibays mezzanine_patched_e-smith-ibays-1.2.0/root/etc/e-smith/web/functions/ibays
--- e-smith-ibays-1.2.0/root/etc/e-smith/web/functions/ibays	2008-07-27 22:55:56.000000000 +0200
+++ mezzanine_patched_e-smith-ibays-1.2.0/root/etc/e-smith/web/functions/ibays	2008-07-27 22:55:12.000000000 +0200
@@ -135,7 +135,7 @@
             <label>PASSWORD_NEW</label>
         </field>
         <field type="password" id="newPassVerify" validation="verifyPasswords">
-            <label>PASSWORD_NEW_VERIFY</label>
+            <label>PASSWORD_VERIFY_NEW</label>
         </field>
         <subroutine src="print_button('SAVE')"/>
     </page>
