diff -Nur -x '*.orig' -x '*.rej' e-smith-ibays-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/ibays mezzanine_patched_e-smith-ibays-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/ibays
--- e-smith-ibays-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/ibays	2008-04-27 18:09:02.000000000 +0200
+++ mezzanine_patched_e-smith-ibays-1.2.0/root/etc/e-smith/locale/en-us/etc/e-smith/web/functions/ibays	2008-04-27 18:04:13.000000000 +0200
@@ -44,30 +44,12 @@
         </trans>
     </entry>
     <entry>
-        <base>NAME</base>
-        <trans>
-            Name
-        </trans>
-    </entry>
-    <entry>
-        <base>DESCRIPTION</base>
-        <trans>
-            Description
-        </trans>
-    </entry>
-    <entry>
         <base>NAME_LABEL</base>
         <trans>
             Information bay name
         </trans>
     </entry>
     <entry>
-        <base>DESCRIPTION_LABEL</base>
-        <trans>
-            Brief description
-        </trans>
-    </entry>
-    <entry>
         <base>USER_ACCESS</base>
         <trans>
             User access via file sharing or user ftp
@@ -119,10 +101,6 @@
         </trans>
     </entry>
     <entry>
-        <base>RESET_PASSWORD</base>
-        <trans>Reset password</trans>
-    </entry>
-    <entry>
         <base>ERROR_WHILE_CREATING_IBAY</base>
         <trans>
             An error occurred while creating the i-bay.
@@ -141,12 +119,6 @@
         </trans>
     </entry>
     <entry>
-        <base>DESCRIPTION_ERROR</base>
-        <trans>
-            Unexpected characters in description.
-        </trans>
-    </entry>
-    <entry>
         <base>CANT_FIND_IBAY</base>
         <trans>
             Can't find account for {$name} (does it exist?)
@@ -191,18 +163,6 @@
         </trans>
     </entry>
     <entry>
-        <base>NEW_PASSWORD</base>
-        <trans>
-            New password
-        </trans>
-    </entry>
-    <entry>
-        <base>NEW_PASSWORD_VERIFY</base>
-        <trans>
-            New password (verify)
-        </trans>
-    </entry>
-    <entry>
         <base>PASSWORD_DESC</base>
         <trans>
             You are about to change the password for the i-bay {$name}.
@@ -284,10 +244,6 @@
         <trans>Entire Internet (password required outside local network)</trans>
     </entry>
     <entry>
-        <base>PASSWORD_VERIFY_ERROR</base>
-        <trans>The passwords you entered did not match.</trans>
-    </entry>
-    <entry>
         <base>INVALID_IBAY_DESCRIPTION</base>
         <trans>Error: unexpected or missing characters in i-bay description</trans>
     </entry>
diff -Nur -x '*.orig' -x '*.rej' e-smith-ibays-1.2.0/root/etc/e-smith/web/functions/ibays mezzanine_patched_e-smith-ibays-1.2.0/root/etc/e-smith/web/functions/ibays
--- e-smith-ibays-1.2.0/root/etc/e-smith/web/functions/ibays	2005-10-06 23:52:28.000000000 +0200
+++ mezzanine_patched_e-smith-ibays-1.2.0/root/etc/e-smith/web/functions/ibays	2008-04-27 18:06:51.000000000 +0200
@@ -129,13 +129,13 @@
         <subroutine src="print_button('REMOVE')" />
     </page>
     <page name="Password" pre-event="turn_off_buttons()" post-event="reset_password()">
-        <title>RESET_PASSWORD</title>
+        <title>PASSWORD_RESET</title>
         <description>PASSWORD_DESC</description>
         <field type="password" id="newPass" validation="nonblank, check_password">
-            <label>NEW_PASSWORD</label>
+            <label>PASSWORD_NEW</label>
         </field>
         <field type="password" id="newPassVerify" validation="verifyPasswords">
-            <label>NEW_PASSWORD_VERIFY</label>
+            <label>PASSWORD_NEW_VERIFY</label>
         </field>
         <subroutine src="print_button('SAVE')"/>
     </page>
diff -Nur -x '*.orig' -x '*.rej' e-smith-ibays-1.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/ibays.pm mezzanine_patched_e-smith-ibays-1.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/ibays.pm
--- e-smith-ibays-1.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/ibays.pm	2008-04-27 18:09:02.000000000 +0200
+++ mezzanine_patched_e-smith-ibays-1.2.0/root/usr/lib/perl5/site_perl/esmith/FormMagick/Panel/ibays.pm	2008-04-27 18:05:18.000000000 +0200
@@ -121,7 +121,7 @@
     my $description = $self->localise('DESCRIPTION');
     my $modify      = $self->localise('MODIFY');
     my $remove      = $self->localise('REMOVE');
-    my $resetpw     = $self->localise('RESET_PASSWORD');
+    my $resetpw     = $self->localise('PASSWORD_RESET');
     my $action_h    = $self->localise('ACTION');
     my @ibays = $accountdb->ibays();
 
@@ -617,7 +617,7 @@
         if (system ("/sbin/e-smith/signal-event", "password-modify", 
             $name) == 0) 
         {
-            $self->success("SUCCESSFULLY_RESET_PASSWORD");
+            $self->success("SUCCESSFULLY_PASSWORD_RESET");
         } else {
             $self->error("ERROR_WHILE_RESETTING_PASSWORD");
         }
@@ -672,7 +672,7 @@
     my $pass1 = $self->{cgi}->param('newPass');
     unless ($pass1 eq $pass2) {
             $self->{cgi}->param( -name => 'wherenext', -value => 'Password' );
-            return "PASSWORD_VERIFY_ERROR";
+            return "PASSWORD_NO_MATCH";
     }
     return "OK";
 }
