diff -Nur -x '*.orig' -x '*.rej' e-smith-ibays-1.1.1/root/etc/e-smith/events/actions/ibay-modify mezzanine_patched_e-smith-ibays-1.1.1/root/etc/e-smith/events/actions/ibay-modify
--- e-smith-ibays-1.1.1/root/etc/e-smith/events/actions/ibay-modify	2005-08-19 01:06:43.036102264 -0600
+++ mezzanine_patched_e-smith-ibays-1.1.1/root/etc/e-smith/events/actions/ibay-modify	2005-08-19 01:05:31.149030760 -0600
@@ -36,11 +36,12 @@
 die "ibayName argument missing" unless defined ($ibayName);
 
 my $accountdb = esmith::AccountsDB->open_ro();
-my $ibay = $accountdb->get($ibayName);
+my $ibay = $accountdb->get($ibayName) or 
+	die "Couldn't find $ibayName record in accounts db\n";
 my $ibayPlugin = esmith::IbayPlugin->load($ibay->prop('Plugin') || 'none');
 
 die "Account $ibayName is not an ibay account; modify ibay event failed.\n"
-    unless ($ibay->type eq 'ibay');
+    unless ($ibay->prop('type') eq 'ibay');
 
 if ($event eq 'ibay-create')
 {
